perception/object_segmentation/elevation_map_loader/src/elevation_map_loader_node.cpp

```diff
@@ -142,33 +78,58 @@ ElevationMapLoaderNode::ElevationMapLoaderNode(const rclcpp::NodeOptions & optio
       "output/elevation_map_cloud", durable_qos);
   }
 
-  std::filesystem::path pointcloud_map_path(
-    this->declare_parameter("pointcloud_map_path", "path_default"));
-  hash_json_ = getPcdMapHashJson(pointcloud_map_path.lexically_normal().string());
+  using std::placeholders::_1;
+  sub_map_hash_ = create_subscription<autoware_external_api_msgs::msg::MapHash>(
+    "/api/autoware/get/map/info/hash", durable_qos,
+    std::bind(&ElevationMapLoaderNode::onMapHash, this, _1));
+  sub_pointcloud_map_ = this->create_subscription<sensor_msgs::msg::PointCloud2>(
+    "input/pointcloud_map", durable_qos,
+    std::bind(&ElevationMapLoaderNode::onPointcloudMap, this, _1));
+  sub_vector_map_ = this->create_subscription<autoware_auto_mapping_msgs::msg::HADMapBin>(
+    "input/vector_map", durable_qos, std::bind(&ElevationMapLoaderNode::onVectorMap, this, _1));
+
+  auto cb = std::bind(&ElevationMapLoaderNode::onTimer, this);
+  auto period =
+    std::chrono::duration_cast<std::chrono::nanoseconds>(std::chrono::duration<double>(0.1));
+  timer_ = std::make_shared<rclcpp::GenericTimer<decltype(cb)>>(
+    get_clock(), period, std::move(cb), get_node_base_interface()->get_context());
+  get_node_timers_interface()->add_timer(timer_, nullptr);
+}
 
-  const auto elevation_map_hash = getMd5Sum(hash_json_.dump());
-  const std::string elevation_map_directory =
-    this->declare_parameter("elevation_map_directory", "path_default");
-  elevation_map_path_ = std::filesystem::path(elevation_map_directory) / elevation_map_hash;
+void ElevationMapLoaderNode::onTimer()
```

Comment:
This function is not called until `clock` is published when `use_sim_time` is `True`.
So `elevation_map` is not created until playing bag.
This is a problem in some first frames in evaluation of Lsim.

Reply:
> This is a problem in some first frames in evaluation of Lsim.

How does the problem affect?

Reply:
We can't evaluate in some first frames because no elevation_map.

Reply:
I fixed this issue by https://github.com/tier4/AutowareArchitectureProposal.iv/pull/740/commits/4c475337e4f8ed272c131cf1ff6a807b3891fa62